src/ol3cesium.js

```diff
@@ -207,6 +224,125 @@ olcs.OLCesium = function(options) {
    * @private
    */
   this.blockCesiumRendering_ = false;
+
+  /**
+   * @type {ol.Feature}
+   * @private
+   */
+  this.trackedFeature_ = null;
+
+  /**
+   * @type {Cesium.Entity}
+   * @private
+   */
+  this.trackedEntity_ = null;
+
+  /**
+   * @type {Cesium.EntityView}
+   * @private
+   */
+  this.entityView_ = null;
+
+  /**
+   * @type {boolean}
+   * @private
+   */
+  this.needTrackedEntityUpdate_ = false;
+
+  /**
+   * @type {!Cesium.BoundingSphere}
+   */
+  this.boundingSphereScratch_ = new Cesium.BoundingSphere();
+
+  var eventHelper = new Cesium.EventHelper();
+  eventHelper.add(this.scene_.postRender, olcs.OLCesium.prototype.updateTrackedEntity_, this);
+};
+
+
+Object.defineProperties(olcs.OLCesium.prototype, {
+  trackedFeature: {
```

Comment:
maybe the compiler is aware of `Object.defineProperties` and don't rename the keys